src/ert/dark_storage/common.py

```diff
@@ -1,31 +1,33 @@
 from typing import Any, Dict, List, Optional, Sequence, Union
+from uuid import UUID
 
 import pandas as pd
 
 from ert.config import EnkfObservationImplementationType
 from ert.libres_facade import LibresFacade
-from ert.storage import EnsembleReader
+from ert.storage import EnsembleReader, StorageReader
 
 
-def ensemble_parameter_names(res: LibresFacade) -> List[str]:
-    return res.gen_kw_keys()
+def ensemble_parameter_names(storage: StorageReader, ensemble_id: UUID) -> List[str]:
+    return storage.get_ensemble(ensemble_id).get_gen_kw_keyset()
 
 
-def ensemble_parameters(res: LibresFacade) -> List[Dict[str, Any]]:
+def ensemble_parameters(
+    storage: StorageReader, ensemble_id: UUID
+) -> List[Dict[str, Any]]:
     return [
         {"name": key, "userdata": {"data_origin": "GEN_KW"}, "labels": []}
-        for key in ensemble_parameter_names(res)
+        for key in ensemble_parameter_names(storage, ensemble_id)
     ]
 
 
-def get_response_names(res: LibresFacade, ensemble: EnsembleReader) -> List[str]:
+def get_response_names(ensemble: EnsembleReader) -> List[str]:
     result = ensemble.get_summary_keyset()
-    result.extend(res.get_gen_data_keys().copy())
+    result.extend(ensemble.get_gen_data_keyset().copy())
```

Comment:
Suggest using the experiment directly for this, as we register all the responses:
```python
list(ensemble.experiment.response_configuration)
```
the main challenge there is that the summary response is named `summary`, we might need to reconsider that.

Reply:
Wont fix in this PR